src/pages/ReimbursementAccount/BankInfo/substeps/Plaid.tsx

```diff
@@ -1,51 +1,47 @@
 import {withOnyx} from 'react-native-onyx';
-import _ from 'underscore';
+import {OnyxEntry} from 'react-native-onyx/lib/types';
 import AddPlaidBankAccount from '@components/AddPlaidBankAccount';
 import Form from '@components/Form';
 import useLocalize from '@hooks/useLocalize';
+import {SubStepProps} from '@hooks/useSubStep/types';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as PlaidDataProps from '@pages/ReimbursementAccount/plaidDataPropTypes';
-import reimbursementAccountDraftPropTypes from '@pages/ReimbursementAccount/ReimbursementAccountDraftPropTypes';
-import {reimbursementAccountPropTypes} from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
 import * as ReimbursementAccountProps from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
-import subStepPropTypes from '@pages/ReimbursementAccount/subStepPropTypes';
-import getDefaultValueForReimbursementAccountField from '@pages/ReimbursementAccount/utils/getDefaultValueForReimbursementAccountField';
 import * as BankAccounts from '@userActions/BankAccounts';
 import * as ReimbursementAccount from '@userActions/ReimbursementAccount';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import {PlaidData, ReimbursementAccountDraft, ReimbursementAccount as TReimbursementAccount} from '@src/types/onyx';
 
-const propTypes = {
+type PlaidOnyxProps = {
     /** Reimbursement account from ONYX */
-    reimbursementAccount: reimbursementAccountPropTypes,
+    reimbursementAccount: OnyxEntry<TReimbursementAccount>;
 
     /** The draft values of the bank account being setup */
-    reimbursementAccountDraft: reimbursementAccountDraftPropTypes,
+    reimbursementAccountDraft: OnyxEntry<ReimbursementAccountDraft>;
 
     /** Contains plaid data */
-    plaidData: PlaidDataProps.plaidDataPropTypes,
-
-    ...subStepPropTypes,
+    plaidData: OnyxEntry<PlaidData>;
 };
 
-const bankInfoStepKeys = CONST.BANK_ACCOUNT.BANK_INFO_STEP.INPUT_KEY;
+type PlaidProps = PlaidOnyxProps & SubStepProps;
 
-const defaultProps = {
-    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
-    reimbursementAccountDraft: {},
-    plaidData: PlaidDataProps.plaidDataDefaultProps,
-};
+const bankInfoStepKeys = CONST.BANK_ACCOUNT.BANK_INFO_STEP.INPUT_KEY;
 
-function Plaid({reimbursementAccount, reimbursementAccountDraft, onNext, plaidData}) {
+function Plaid({
+    reimbursementAccount = ReimbursementAccountProps.reimbursementAccountDefaultProps,
+    reimbursementAccountDraft,
+    onNext,
+    plaidData = PlaidDataProps.plaidDataDefaultProps,
+}: PlaidProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
     const isFocused = useIsFocused();
 
-    const validate = useCallback((values) => {
-        const errorFields = {};
+    const validate = useCallback((values: {acceptTerms: boolean}) => {
+        const errorFields: Record<string, string> = {};
```

Comment:
```suggestion
        const errorFields: Errors = {};
```

You can use `Errors` from `src/types/onyx/OnyxCommon.ts`